aidies-project/client/src/components/LoginPanel.js

```diff
@@ -42,16 +42,15 @@ const LoginPanel = (props) => {
     const [email, setEmail] = useState('demo@example.com');
     const [password, setPassword] = useState('password');
     const dispatch = useDispatch();
-    const user = useSelector(state => state.authentication.user.id);
+    const userId = useSelector(state => state.authentication.user.id);
     const location = useSelector(state => state.campaign.previousLocation);
-    const history = useHistory();
     const classes = useStyles();
 
     const handleSubmit = (e) =>{
         e.preventDefault();
         dispatch(login(email, password));
     }
-    if (user!== null) {
+    if (userId!== null) {
```

Comment:
I would suggest cleaning up the conditionals to make the code look a little cleaner!

```js
if (userId) {
    return (location) ? <Redirect to={location} /> : <Redirect to="/" />;
}
```